fe/fe-core/src/main/java/org/apache/doris/planner/external/HiveScanNode.java

```diff
@@ -310,4 +299,28 @@ private void genSlotToSchemaIdMap() {
         }
         params.setSlotNameToSchemaPos(columnNameToPosition);
     }
+
+    @Override
+    public boolean pushDownAggNoGrouping(FunctionCallExpr aggExpr) {
+        TFileFormatType fileFormatType;
+        try {
+            fileFormatType = getFileFormatType();
+        } catch (UserException e) {
+            throw new RuntimeException(e);
+        }
+
+        String aggFunctionName = aggExpr.getFnName().getFunction();
+        if (aggFunctionName.equalsIgnoreCase("COUNT") && fileFormatType == TFileFormatType.FORMAT_PARQUET) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean pushDownAggNoGroupingCheckCol(FunctionCallExpr aggExpr, Column col) {
```

Comment:
For external table, always return false.

Reply:
when you use `select count(*)`statement   ,`pushDownAggNoGroupingCheckCol` will not be executed .
if you use `select count(a)` statement , `pushDownAggNoGroupingCheckCol` will check col a.